pymc3/tests/test_model_graph.py

```diff
@@ -14,6 +14,8 @@ def radon_model():
     floor_measure = np.random.randint(0, 2, size=n_homes)
     log_radon = np.random.normal(1, 1, size=n_homes)
 
+    floor_measure = th.shared(floor_measure)
```

Comment:
thanks for keeping at it -- I think you just need `import theano as th` or similar at the top of this file.

Reply:
Yikes! Thanks. I had it in the previous PR. Fixed.